backend/test/modules/tournament-details/presentation/rest-api/TournamentDetailsRestApiModule.spec.ts

```diff
@@ -0,0 +1,84 @@
+import { testModuleRestApi } from '../../../../test-support/shared/presentation/rest-api/TestModuleRestApi';
+import { StatusCodes } from 'http-status-codes';
+import { CommandPublisherMock } from '../../../../test-support/shared/core/CommandPublisherMock';
+import { CommandResult } from '../../../../../src/shared/core/application/command/CommandResult';
+import { QueryPublisherMock } from '../../../../test-support/shared/core/QueryPublisherMock';
+import { TournamentDetailsRestApiModule } from '../../../../../src/modules/tournament-details/presentation/rest-api/TournamentDetailsRestApiModule';
+import { AddTournamentDetails } from '../../../../../src/modules/tournament-details/core/application/command/AddTournamentDetails';
+import { TournamentDetails } from '../../../../../src/modules/tournament-details/core/domain/TournamentDetails';
+import { TournamentName } from '../../../../../src/modules/tournament-details/core/domain/TournamentName';
+import { FindAllTournamentDetails } from '../../../../../src/modules/tournament-details/core/application/query/FindAllTournamentDetails';
+
+describe('Tournament Details REST API', () => {
+  it('POST /rest-api/tournament-details | when command success', async () => {
+    //Given
+    const commandPublisher = CommandPublisherMock(CommandResult.success());
+    const { agent } = testModuleRestApi(TournamentDetailsRestApiModule, { commandPublisher });
+
+    //When
+    const { body, status } = await agent
+      .post('/rest-api/tournament-details')
+      .send({ tournamentId: 'SampleTournamentId', tournamentName: 'SampleName' });
+
+    //Then
+    expect(commandPublisher.executeCalls).toBeCalledWith(
+      new AddTournamentDetails({ tournamentId: 'SampleTournamentId', tournamentName: 'SampleName' }),
+    );
+    expect(status).toBe(StatusCodes.CREATED);
+    expect(body).toStrictEqual({ tournamentId: 'SampleTournamentId', tournamentName: 'SampleName' });
```

Comment:
Nie ma po co tego zwracac, przeciez to wysylamy w requescie. Wystarczy, ze dostaniemy tylko status.

Reply:
już mi na to kiedyś zwróciłeś uwagę... nic się nie uczę na błędach :P